### PR TITLE
+cowsay

### PR DESCRIPTION
**Pull request by nakajima**
Adds cowsay.

### CODE REVIEW

File: projects/github.com/cowsay-org/cowsay/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ref: ${{version.tag}}
+
+display-name: cowsay
+
+versions:
+  github: cowsay-org/cowsay
+
+platforms:
+  - darwin
+  - linux/x86-64
```
Comment (jhheider) [inline projects/github.com/cowsay-org/cowsay/package.yml]: is this intentional? we should either have a comment as to why it doesn't work on linux/aarch64, or remove this stanza if it does.

File: projects/github.com/cowsay-org/cowsay/package.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+dependencies:
+  perl.org: '*'
+
+build:
+  script:
+    - run: cpan File::Find
```
Comment (jhheider) [inline projects/github.com/cowsay-org/cowsay/package.yml]: we should use `cpanmin` to install it in the prefix. see https://github.com/pkgxdev/pantry/blob/main/projects/gnu.org/help2man/package.yml for example.

File: projects/github.com/cowsay-org/cowsay/package.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: git+https://github.com/cowsay-org/cowsay.git
+  ref: ${{version.tag}}
```
Comment (jhheider) [inline projects/github.com/cowsay-org/cowsay/package.yml]: is it the case that using the packaged sources doesn't work? i'm fine if that's the case, but i'd be surprised for a perl package.

File: projects/github.com/cowsay-org/cowsay/package.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  perl.org: '*'
+
+build:
+  script:
+    - run: cpan File::Find
+      if: linux
+    - make install prefix={{prefix}}
+
+provides:
+  - cowsay
```
Comment (jhheider) [inline projects/github.com/cowsay-org/cowsay/package.yml]: this should be `- bin/cowsay`

**Review by jhheider — CHANGES_REQUESTED** [review]

File: projects/github.com/cowsay-org/cowsay/package.yml (hunk 3 above)
Comment (nakajima) [reply projects/github.com/cowsay-org/cowsay/package.yml]: Nope, updated.

File: projects/github.com/cowsay-org/cowsay/package.yml (hunk 1 above)
Comment (nakajima) [reply projects/github.com/cowsay-org/cowsay/package.yml]: Nope, I think that's just what was there in the generated file.

**Comment by nakajima** [comment]
@jhheider Thanks for the review, it's my first PR here so I still don't quite have the lay of the land. I think I took care of everything!

**Review by jhheider — APPROVED** [review]
looks good!

**Pull request merged**